x-pack/plugins/cases/public/api/utils.ts

```diff
@@ -26,6 +26,8 @@ import type {
   ResolvedCase,
 } from '../containers/types';
 
+const UUID_REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
I wasn't aware that the beginning of the 3rd and 4th portions(`[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}`) was limited to certain characters! Is this limited to a specific version of uuid?

Also, what about capitals?

Reply:
@js-jankisalvi See below in `convertToCamelCase`:

```js
let convertedKey = key;
// Do NOT convert UUID keys to camelCase
if (!UUID_REGEX.test(key)) {
  convertedKey = camelCase(key);
}
```

However, I think @adcoelho is right that we can just use `uuid.validate()` instead.

Reply:
@adcoelho Note that `uuid.validate()`[ uses basically the same regex as this one here](https://github.com/uuidjs/uuid/blob/main/src/regex.ts) but I agree we might as well import it from the third party lib we're already using